src/source/source_cache.js

```diff
@@ -324,59 +324,64 @@ class SourceCache extends Evented {
     }
 
     /**
-     * Recursively find children of the given tile (up to maxCoveringZoom) that are already loaded;
-     * adds found tiles to retain object; returns true if any child is found.
+     * Retain children of the given set of tiles (up to maxCoveringZoom) that are already loaded;
      */
-    _findLoadedChildren(tileID: OverscaledTileID, maxCoveringZoom: number, retain: {[any]: OverscaledTileID}): boolean {
-        let found = false;
-
+    _retainLoadedChildren(
+        tiles: {[any]: OverscaledTileID},
```

Comment:
> can the any be replaced with number here?

Yes, but I'd leave this for a separate PR, since improving types for all objects that use tile IDs as keys is a pretty big change. Currently it's either `any` or `string | number` throughout the code because when you iterate with `for (const id in obj)`, `id` is always a string, and converting it everywhere is cumbersome.